.github/workflows/main.yml

```diff
@@ -46,35 +46,37 @@ jobs:
         run: ant -f platform/core.network test
 
       - name: Test PHP modules
+        env:
+          OPTS: -quiet -Dcluster.config=php -Djavac.compilerargs=-nowarn -Dbuild.compiler.deprecation=false -Dtest-unit-sys-prop.ignore.random.failures=true
```

Comment:
@KacerCZ Please. Remove `-quiet -Dcluster.config=php -Djavac.compilerargs=-nowarn -Dbuild.compiler.deprecation=false` params, it isn't required.

Reply:
Removed.